### PR TITLE
Add the ability to download WebDriver on the new MicrosoftEdge

### PR DESCRIPTION
**Pull request by loly89**
Creating another wpt product 'chromiumedge' for the new Microsoft Edge since it's totally different compared to the old one.

The ability to download and install the new MicrosoftEdge browser will be coming soon. 
As of now, users will need to download, install MicrosoftEdge insider version and add the installed location to PATH system variable.

### CODE REVIEW

**Comment by gsnedders** [comment]
Do we want this to be `chromiumedge` or `edgechromium`? I think I'd rather the latter because then it starts with `edge`, which is probably more expected. At a quick glance, this otherwise looks good.

**Comment by loly89** [comment]
> Do we want this to be `chromiumedge` or `edgechromium`? I think I'd rather the latter because then it starts with `edge`, which is probably more expected. At a quick glance, this otherwise looks good.

I've changed the product to be 'edgechromium'. Do you have any other feedback? :)

**Comment by lukebjerring** [comment]
@Hexcles  FYI - might be worth double checking the handling of `edgechromium` in wpt.fyi

File: tools/wpt/browser.py [hunk 1]
```diff
@@ -632,6 +633,50 @@ def version(self, binary=None, webdriver_binary=None):
...
+            raise ValueError("Only Windows platform is currently supported")
+
+        if dest is None:
+            dest = os.pwd
+
+        platform = "x64" if uname[4] == "x86_64" else "x86"
+
+        url = "https://az813057.vo.msecnd.net/webdriver/msedgedriver_%s/msedgedriver.exe" % platform
+        self.logger.info("Downloading MSEdgeDriver from %s" % url)
+        urllib.urlretrieve(url, os.path.join(dest, "msedgedriver.exe"))
```
Comment (gsnedders) [inline tools/wpt/browser.py]: Please use `requests` for the download (see elsewhere in this file)

File: tools/wpt/run.py [hunk 2]
```diff
@@ -327,6 +327,29 @@ def setup_kwargs(self, kwargs):
                 raise WptrunError("Unable to locate or install operadriver binary")
 
 
+class EdgeChromium(BrowserSetup):
+    name = "MicrosoftEdge"
```
Comment (gsnedders) [inline tools/wpt/run.py]:
```suggestion
    name = "EdgeChromium"
```

…rather than having multiple things claiming to be MicrosoftEdge.

**Review by gsnedders — CHANGES_REQUESTED** [review]
Couple of small issues, but that's all. 🙂

File: tools/wpt/run.py (hunk 2 above)
Comment (loly89) [reply tools/wpt/run.py]: Due to some legal issues, we have to keep the same name for the browser name. Also, MSEdgeDriver is checking the browser name against "MicrosoftEdge" in the capability, so changing that will make the WebDriver not working correctly.

Comment (gsnedders) [reply tools/wpt/run.py]: Okay.

**Review by gsnedders — APPROVED** [review]

**Pull request merged**